packages/ERTP/src/issuer.js

```diff
@@ -141,6 +142,10 @@ function makeIssuerKit(
     return purse;
   });
 
+  const add = (left, right) => amountMath.add(left, right, brand);
+  const empty = amountMath.makeEmpty(amountMathKind);
+  const coerce = allegedAmount => amountMath.coerce(allegedAmount, brand);
```

Comment:
Nice.

Since you're factoring out these others anyway, it would be surprising not to factor out `subtract` as part of the pile. Also pleasant.